hisensetv/__init__.py

```diff
@@ -226,6 +239,60 @@ def send_key(self, keyname: str):
         """
         self._call_service(service="remote_service", action="sendkey", payload=keyname)
 
+    def launch_app(self, app: str):
+
+        if app == "amazon":
+            launch = {"name": "Amazon", "urlType": 37, "storeType": 0, "url": "amazon"}
+
+        if app == "netflix":
+            launch = {
+                "name": "Netflix",
+                "urlType": 37,
+                "storeType": 0,
+                "url": "netflix",
+            }
+
+        elif app == "youtube":
+            launch = {
+                "name": "YouTube",
+                "urlType": 37,
+                "storeType": 0,
+                "url": "youtube",
+            }
+
+        else:
+            pass
+
+        self._call_service(service="ui_service", action="launchapp", payload=launch)
+
+    def change_source(self, id: str):
+
+        if id == "0":
+            source = {"sourceid": "0"}
+
+        if id == "1":
+            source = {"sourceid": "1"}
+
+        if id == "2":
+            source = {"sourceid": "2"}
+
+        if id == "3":
+            source = {"sourceid": "3"}
+
+        if id == "4":
+            source = {"sourceid": "4"}
+
+        if id == "5":
+            source = {"sourceid": "5"}
+
+        if id == "6":
+            source = {"sourceid": "6"}
+
+        if id == "7":
+            source = {"sourceid": "7"}
```

Comment:
This can be shortened:
```suggestion
    def change_source(self, id: str):
        payload = {"sourceid": id}
        self._call_service(service="ui_service", action="changesource", payload=payload)
```